tests/snuba/search/test_backend.py

```diff
@@ -2051,7 +2059,209 @@ def test_query(query):
             test_query(f"{key}:{val}")
 
 
-class CdcEventsSnubaSearchTest(TestCase, SnubaTestCase):
+class EventsTransactionsSnubaSearchTest(SharedSnubaTest):
+    @property
+    def backend(self):
+        return EventsDatasetSnubaSearchBackend()
+
+    def setUp(self):
+        super().setUp()
+        self.base_datetime = (datetime.utcnow() - timedelta(days=3)).replace(tzinfo=pytz.utc)
+
+        transaction_event_data = {
+            "level": "info",
+            "message": "ayoo",
+            "type": "transaction",
+            "culprit": "app/components/events/eventEntries in map",
+            "contexts": {"trace": {"trace_id": "b" * 32, "span_id": "c" * 16, "op": ""}},
+        }
+
+        def hack_pull_out_data(jobs, projects):
+            _pull_out_data(jobs, projects)
+            for job in jobs:
+                job["event"].groups = [self.group]
+            return jobs, projects
+
+        with mock.patch("sentry.event_manager._pull_out_data", hack_pull_out_data):
+            transaction_event = self.store_event(
+                data={
+                    **transaction_event_data,
+                    "event_id": "a" * 32,
+                    "timestamp": iso_format(before_now(minutes=1)),
+                    "start_timestamp": iso_format(before_now(minutes=1)),
+                    "tags": {"my_tag": 1},
+                },
+                project_id=self.project.id,
+            )
+
+        transaction_event.group = self.group
+        self.perf_group_1 = Group.objects.get(id=transaction_event.group_id)
+        self.perf_group_1.update(type=GroupType.PERFORMANCE_SLOW_SPAN.value)
+
+        with mock.patch("sentry.event_manager._pull_out_data", hack_pull_out_data):
+            transaction_event = self.store_event(
+                data={
+                    **transaction_event_data,
+                    "event_id": "b" * 32,
+                    "timestamp": iso_format(before_now(minutes=1)),
+                    "start_timestamp": iso_format(before_now(minutes=1)),
```

Comment:
Let's make the timestamps different here so that perf_group_1 and perf_group_2 sort deterministically